packages/react/package.json

```diff
@@ -18,17 +18,17 @@
     "dist/"
   ],
   "devDependencies": {
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
+    "@types/react": "^17.0.3",
+    "@types/react-dom": "^17.0.3",
+    "react": "^16.14.0",
```

Comment:
Dit is een downgrade naar React 16, dit is de React versie die ze bij DSO gebruiken